### ISSUE TITLE
The console should be run against an accessibility checking tool

### ISSUE THREAD

**Issue by msecret** [18F/cg-deck#325]
In order to ensure the console site work's on as many devices as possible, it should be run through an accessibility checking tool such as [HTML code-sniffer](http://squizlabs.github.io/HTML_CodeSniffer/).

- [ ] Check each page of the site with [HTML code-sniffer](http://squizlabs.github.io/HTML_CodeSniffer/).
- [ ] All messages of the "alert" or "warning" level should be filed as top priority tasks to be fixed before release to production
- [ ]  All messages of the "info" level should be investigated and if valid, should be logged as an issue.

**Comment by msecret** [comment]
I scanned the following pages with all potential UI open:

- `/#/`
- `/#/org/ooooo/marketplace`
- `/#/org/ooooo/spaces/sssss`
- `/#/org/ooooo/spaces/sssss/services`
- `/#/org/ooooo/spaces/sssss/users`
- `/#/org/ooooo/spaces/sssss/apps/aaaaa`

Anything I could've missed?

**Comment by jeremiak** [comment]
Does "all potential UI open" mean that you removed authentication somehow?

**Comment by jeremiak** [comment]
Are all resulting issues in an epic or tagged with a label? It looks like it might be the "accessibility" label but want to confirm.

**Comment by msecret** [comment]
Yes it's that

**Comment by jeremiak** [comment]
Great, this is done. We're working our way through the issues tagged with the "accessibility" label
